airport/serializers.py

```diff
@@ -0,0 +1,242 @@
+from datetime import datetime, timedelta
+
+from django.db import transaction
+from rest_framework import serializers
+from rest_framework.exceptions import ValidationError
+
+from airport.models import (Airport,
+                            Airplane,
+                            Airline,
+                            Crew,
+                            Flight,
+                            Ticket,
+                            Order)
+from utilities.get_weather import get_weather
+
+
+class AirportSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Airport
+        fields = "__all__"
+
+
+class AirportDetailSerializer(AirportSerializer):
+    current_weather = serializers.SerializerMethodField(
+        method_name="get_airport_weather"
+    )
+
+    class Meta:
+        model = Airport
+        fields = (
+            "id",
+            "icao_designator",
+            "closest_big_city",
+            "current_weather",
+        )
+
+    @staticmethod
+    def get_airport_weather(obj):
+        weather = get_weather(obj.closest_big_city)
+        return weather
+
+
+class AirlineSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Airline
+        fields = "__all__"
+
+
+class AirplaneSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Airplane
+        fields = "__all__"

```

Comment:
better keep your code in one-style, like you do it below
i mean this:  fields = "__all__" > fields = ("id", "image")